layer/nrlf/nrlf/core/tests/test_event_parsing.py

```diff
@@ -19,17 +19,16 @@ def test_parsing_body_from_event(event, expected_body):
 
 
 @pytest.mark.parametrize(
-    "event",
+    ["event", "error_message"],
     [
-        make_aws_event(body=None),
-        make_aws_event(body=12345),
-        make_aws_event(body="12345"),
-        make_aws_event(body=True),
+        [make_aws_event(body="12345"), "Body is not expected json type"],
```

Comment:
minor comment is not that we're not testing the other error message here, could add

```suggestion
        [make_aws_event(body="{not json]"), "Body is not valid json"],
        [make_aws_event(body="12345"), "Body is not expected json type"],            
```